testing/features/step_definitions/pagination.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+Given("an Example Pagination component is on the page") do
+  @component = Pagination::Example.new.tap(&:load)
+end
+
+Given("a Paging Info Pagination component is on the page") do
+  @component = Pagination::PagingInfo.new.tap(&:load)
+end
+
+Then("the current page is highlighted") do
+  if chrome? || mobile_phone?
+    expect(@component.current_page.native.css_value("background-color")).to eq("rgba(1, 129, 118, 1)")
```

Comment:
We shouldn't text colours like this in feature tests - the visual regression tests should be sufficient.

Reply:
Agree with you, I don't like to assert this either, but the steps say:
`Then the current page is highlighted`
This was the way I found to assert if the current_page was *highlighted*.

I'm going to change the step to something like `Then current page displayed` and then `expect(@component).to have_current_page` in the step-definition.